Makefile

```diff
@@ -48,6 +49,12 @@ bin/atc: src/log/*.go src/atc/*.go
 	@mkdir -p bin
 	$(BUILD) -o $@ ${SRC}/atc
 
+jsx: src/react/jsx/*.js
+	@cd src/react && $(NPM) run build-js
```

Comment:
It's up to you whether or not to remove the `@`, I just thought logging the npm command would be helpful. I don't care so much about the `cd`. It's unfortunate there's not an easy way around this.